src/react/DevicesPage.jsx

```diff
@@ -42,6 +43,7 @@ export default ({ astarte, history }) => {
   const [activePage, setActivePage] = useState(0);
   const [maxPage, setMaxPage] = useState(0);
   const [cachedPages, setCachedPages] = useState([]);
+  const pageAlerts = useAlerts();
```

Comment:
You are not displaying `<pageAlerts.Alerts />` to the user. Forgot to add it in the JSX?